src/Dtos/FakturowniaDto.php

```diff
@@ -42,11 +42,11 @@ public function __construct(Order $order)
         $this->setSellDate($this->now->format('Y-m-d'));
         $this->setIssueDate($this->now->format('Y-m-d'));
         $this->setBuyerEmail($order->nip ?? '');
-        $this->setBuyerName($order->first_name ?? $order->user->first_name . " " . $order->last_name ?? $order->user->last_name);
-        $this->setBuyerTaxNo($order->nip ?? '');
-        $this->setBuyerPostCode($order->zip_code ?? '');
-        $this->setBuyerCity($order->city ?? '');
-        $this->setBuyerStreet($order->street ?? '');
+        $this->setBuyerName($order->client_name ?? $order->client_company ?? ($order->user->first_name . " " . $order->last_name) ?? '');
+        $this->setBuyerTaxNo($order->client_taxid ?? '');
+        $this->setBuyerPostCode($order->client_postal ?? '');
+        $this->setBuyerCity($order->client_city ?? '');
+        $this->setBuyerStreet($order->client_street ?? '');
```

Comment:
There is now additional `client_street_number` field in Orders table

Reply:
done